data-products/src/main/scala/org/sunbird/analytics/audit/ScoreMetricMigrationJob.scala

```diff
@@ -44,41 +43,61 @@ object ScoreMetricMigrationJob extends optional.Application with IJob with BaseR
   }
 
   // $COVERAGE-ON$
-  def migrateData(session: SparkSession): DataFrame = {
+  def migrateData(session: SparkSession, jobConfig: JobConfig): DataFrame = {
+    val batchIds: List[String] = jobConfig.modelParams.get.getOrElse("batchId", List()).asInstanceOf[List[String]]
```

Comment:
Please have comments in the code on why we required these configs? what are these usage?